loader/0002-quilt.mod.json.md

```diff
@@ -0,0 +1,419 @@
+    * The field must be static.
+    * The type of the field must be assignable from the field's class.
+    * Example: `example.mod.MainModClass::THE_INSTANCE`
+    * If there is ambuguity with a method's name, an exception will be thrown.
+* A method inside of a class.
+    * The value must contain a fully qualified binary name to the class followed by `::` and a method name.
+    * The method must be capable to implement the entrypoint type as a method reference. Generally this means classes which are functional interfaces.
+    * Constructor requirement varies based on the method being static or instance level:
+        * A static method does not require a public no-argument constructor.
+        * An instance method requires a public no-argument constructor.
+    * Example: `example.mod.MainModClass::init`
+    * If there is ambuguity with a fields's name or other method, an exception will be thrown.
+
+#### Other notations
+
+Some language providers may extend the capabilities of the JVM language notation or provide additional notations.
+For notation rules for other languuage adapters, consult the capabilities of the language adapter.
+
+### The `plugins` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of loader plugins. A plugin is an object with the following keys:
+* adapter — Language adapter to use for this plugin
+* value — Points to an implementation of the `LoaderPlugin` interface. Can be in either of the following forms:
+    * `my.package.MyClass` — A class to be instantiated and used
+    * `my.package.MyClass::thing` — A static field containing an instance of a `LoaderPlugin`
+
+If a plugin does not need to specify a language adapter other than the default language adapter, the plugin can be represented simply as the value string instead.
+
+### The `jars` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+A list of paths to nested JAR files to load, relative to the root directory inside of the mods JAR.
+
+### The `language_adapters` field
+| Type   | Required |
+|--------|----------|
+| Object | False    |
+
+A collection of `key: value` pairs, where each key is the namespace of a language adapter and the value is an implementation of the `LanguageAdapter` interface.
+
+### The `depends` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of [dependency object](#dependency-objects)s. Defines mods that this mod will not function without.
+
+### The `breaks` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of [dependency object](#dependency-objects)s. Defines mods that this mod either breaks or is broken by.
+
+### The `repositories` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+A list of repository url strings where dependencies can be looked for in addition to Quilt's central maven repository.
```

Comment:
Does this only support Maven?

Reply:
Yes. What else would it support?

Reply:
It should clarify it then

Reply:
I feel like it's relatively self-explainatory

Reply:
How so? You might be using an Ivy repo instead (as is easier to do if you use Github as a host)

Reply:
Maven is mentioned many times throughout the document, and even later in this same sentence. I can add some clarification, but it doesn't really seem necessary.